butterknife/src/main/java/butterknife/ButterKnife.java

```diff
@@ -83,7 +85,7 @@ private ButterKnife() {
     throw new AssertionError("No instances.");
   }
 
-  /** An unbinder contract that can be bind with {@link butterknife.Unbinder}. */
+  /** An unbinder contract that will unbind views when called. */
   @SuppressWarnings("unused") // Used by generated code.
   public interface ViewUnbinder<T> {
```

Comment:
The `unbind` method takes no argument. And method parameter types are not covariant so you wouldn't be able to override with a more specific argument type anyway.